packages/core/tests/oob.test.ts

```diff
@@ -658,12 +658,11 @@ describe('out of band', () => {
     })
 
     test('throw an error when a did is used in the out of band message', async () => {
-      const { message } = await faberAgent.credentials.createOffer(credentialTemplate)
-      const { outOfBandInvitation } = await faberAgent.oob.createInvitation({
-        ...issueCredentialConfig,
-        messages: [message],
+      const outOfBandInvitation = new OutOfBandInvitation({
+        label: 'test-connection',
+        handshakeProtocols: [HandshakeProtocol.DidExchange],
+        services: ['somedid'],
       })
-      outOfBandInvitation.services = ['somedid']
```

Comment:
This test is not applicable anymore?

Reply:
true, this case is not applicable anymore